scripts/pyperf_build.sh

```diff
@@ -5,7 +5,7 @@
 #
 set -euo pipefail
 
-git clone --depth 1 -b v1.2.6 https://github.com/Granulate/bcc.git && cd bcc && git reset --hard 6a995642d12287f26cbb97edd3d29e62a5fde337
+git clone --depth 1 -b upgrade-bcc-reference https://github.com/marcin-ol/bcc.git && cd bcc && git reset --hard 863d545
```

Comment:
`863d545` is still the commit from Dec 2022, I recall you said you upgraded it? Or there was no need to update BCC again?

Reply:
Our reference is still bcc 0.25.
We might upgrade to 0.27, but I believe it doesn't bring us anything immediately useful besides perhaps this:
- [Implement symbol lookup in zipped libraries. #4440](https://github.com/iovisor/bcc/pull/4440)

I guess we needn't upgrade further atm. 
But I might try to check the upgrade anyway.

Reply:
Not a must. Can be done a separate PR.